fairlearn/metrics/__init__.py

```diff
@@ -17,13 +20,34 @@
 
 # Classification metrics
 group_specificity_score = make_group_metric(specificity_score)
+"""A grouped metric for the specificity score
+"""
+
 group_miss_rate = make_group_metric(miss_rate)
+"""A grouped metric for the miss rate
+"""
+
 group_fallout_rate = make_group_metric(fallout_rate)
+"""A grouped metric for the fallout rate
+"""
 
 # Regression metrics
 group_max_error = make_group_metric(skm.max_error)
+"""A grouped wrapper around the sklearn.metrics.max_error routine
+"""
+
 group_mean_absolute_error = make_group_metric(skm.mean_absolute_error)
+"""A grouped wrapper around the sklearn.metrics.mean_absolute_error
```

Comment:
Interesting that the doc comes *after* the definition and not before :O

Reply:
Indeed.